CMakeLists.txt

```
@@ -208,7 +208,7 @@ if(STRIP_DEBUGINFO)
         VERBATIM
     )
 endif()
-install (TARGETS ruri DESTINATION /usr/bin/)
+install (TARGETS ruri DESTINATION)
```

This doesn't make sense.  Have you tested this change locally?

---

> This doesn't make sense. Have you tested this change locally?

```
user@localhost ~/ruri (main)> ninja
[0/1] Re-running CMake...
-- Using ccache: /usr/bin/ccache
-- Compiler supports flag: -flto=auto
-- Compiler supports flag: -pie
-- Compiler supports flag: -fstack-protector-all
-- Compiler supports flag: -fdata-sections
-- Compiler supports flag: -fno-omit-frame-pointer
-- Compiler supports flag: -fno-stack-protector
-- Compiler supports flag: -ftrivial-auto-var-init=pattern
-- Compiler supports flag: -fstack-clash-protection
-- Compiler supports flag: -Wno-unused-result
CMake Warning at CMakeLists.txt:136 (message):
  Compiler does not support flag: -mshstk


-- Compiler supports flag: -ffunction-sections
-- Compiler supports flag: -Wl,--gc-sections
-- Compiler supports flag: -Wl,--disable-new-dtags
-- Compiler supports flag: -Wl,--build-id=sha1
-- Compiler supports flag: -Wl,-z,norelro
-- Compiler supports flag: -Wl,-z,execstack
-- Compiler supports flag: -Wall
-- Compiler supports flag: -Wextra
-- Compiler supports flag: -Wconversion
-- Compiler supports flag: -pedantic
-- Compiler supports flag: -pipe
-- Compiler supports flag: -Wl,--strip-all
-- Configuring done (9.9s)
-- Generating done (0.0s)
-- Build files have been written to: /home/user/ruri
ninja: no work to do.
user@localhost ~/ruri (main)> sudo ninja install
[0/1] Install the project...
-- Install configuration: ""
-- Installing: /usr/local/bin/ruri
```

---

This would already work I think.

```suggestion
install (TARGETS ruri)
```